src/mockServiceWorker.js

```diff
@@ -108,6 +109,10 @@ self.addEventListener('fetch', function (event) {
     return
   }
 
+  if (!activeClientIds.has(clientId)) {
```

Comment:
Does putting this check earlier than `handleRequest` also has some effect?

We do check it right here:
https://github.com/mswjs/msw/blob/827a5dc0552f0c07822c0b178c85d6bca3d3bd6c/src/mockServiceWorker.js#L130

Reply:
This still returns a response if the active client isnt registered

Reply:
Oh, I see! That's a good point. Thanks for elaborating on this one ❤️